src/translations/en.json

```diff
@@ -1,7 +1,14 @@
 {
   "translation": {
     "ERRORS": {
-      "FIELD_REQUIRED": "Field is required"
+      "FIELD_REQUIRED": "Field is required",
+      "NUMBER": "Password shoul have at least one digit",
+      "UPPER_CASE": "Password should have at least one uppercase letter",
```

Comment:
sorry, I've just seen, that you use bad translations. you have to use reusable messages. This message we can use only for passwords and cannot reuse it

Reply:
fix all English and Ukrainian messages in both files